assets/js/src/core/components/tag-list/tag-list.styles.ts

```diff
@@ -0,0 +1,28 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { createStyles } from 'antd-style'
+
+interface StylesProps {
+  itemRowGap?: number
+  itemColGap?: number
+}
+
+export const useStyles = createStyles(({ css, token }, props: StylesProps) => {
+  return {
+    tagListGroup: css`
+      row-gap: ${props.itemRowGap ?? token.Tag.rowGapXXS}px;
+      column-gap: ${props.itemColGap ?? token.Tag.colGapXXS}px;
```

Comment:
I would prefer when we align this with our general spacing values like we have it in the Flex or Space Component.

We also could think of making a modification on the flex component.
So that we have the possibility to only use a normal string value, like we already have it right now.
And in addition you also could pass something like that:

``` TypeScript
<Flex gap={{ x: 'small', y: 'mini' }}>
    ...list of tags.
</Flex>
```

What do you think?